client/app/scripts/superdesk/editor/editor.js

```diff
@@ -360,9 +389,17 @@ angular.module('superdesk.editor', [])
 
                     editorElem.on('input blur', function(event) {
                         $timeout.cancel(updateTimeout);
-                        updateTimeout = $timeout(updateModel, 300, false);
+                        updateTimeout = $timeout(function() { updateModel(event); }, 300, false);
+                    });
+                    editorElem.on('keyup', function(event) {
+                        if (event.ctrlKey) {
+                            if (event.keyCode === 90) {
+                                doUndo();
+                            } else if (event.keyCode === 89) {
```

Comment:
I suggest creating an alias for these key codes for better readability, perhaps something like the following example:
```js
var KEY_CODES = Object.freeze({
    Y: 89,
    X: 90
});

if (event.keyCode === KEY_CODES.X) { ... }
```

I would also consider putting the key codes definition into an Angular constant or something, because they are used at multiple places throughout the app...